PhysicsTools/PatAlgos/interface/SoftMuonMvaEstimator.h

```diff
@@ -0,0 +1,44 @@
+#ifndef __PhysicsTools_PatAlgos_SoftMuonMvaEstimator__
+#define __PhysicsTools_PatAlgos_SoftMuonMvaEstimator__
+#include "DataFormats/MuonReco/interface/Muon.h"
+#include "DataFormats/PatCandidates/interface/Muon.h"
+#include "TMVA/Reader.h"
+
+namespace pat {
+class SoftMuonMvaEstimator{
+public:
+  SoftMuonMvaEstimator();
+  void initialize(std::string weightsfile);
+  void computeMva(const pat::Muon& imuon);
+  float mva() const {return mva_;}
+  private:
+  TMVA::Reader tmvaReader_;
+  bool initialized_;
+  float mva_;
+
+  // MVA Spectator
+  float pID_;
+  float pt_;
+  float eta_;
+  float MomID_;
```

Comment:
start local variables and data members with lower case letter. Capitalization is reserved for class names.

Reply:
I'll change it in the next commit.